### PR TITLE
rel: v2.4.0

### PR DESCRIPTION
**Pull request by jharley**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,77 @@
...
+- maint: delete workflows for old board (#144) | [Vera Reynolds](https://github.com/vreynolds)
+- Bump django from 4.1 to 4.1.2 in /examples/django_simple (#146) | [dependabot[bot]](https://github.com/dependabot)
+- Bump django from 4.1 to 4.1.2 in /examples/django_response_time (#147) | [dependabot[bot]](https://github.com/dependabot)
+- Bump django from 4.1 to 4.1.2 in /examples/django_dynamic_fields (#145) | [dependabot[bot]](https://github.com/dependabot)
+- Be more specific on required versions of urllib3 #141 (#142) | [Emily Ashley](https://github.com/emilyashley)
+- maint: add release file (#143) | [Vera Reynolds](https://github.com/vreynolds)
+- maint: add new project workflow (#140) | [Vera Reynolds](https://github.com/vreynolds)
+- update releasing notes (#138) | [Emily Ashley](https://github.com/emilyashley)
+
+
```
Comment (JamieDanielson) [inline CHANGELOG.md:74]:
```suggestion
- update releasing notes (#138) | [Emily Ashley](https://github.com/emilyashley)

```
nit

**Review by JamieDanielson — COMMENTED** [review]
Whew it's been a while 😅 ! Looks like we need to run a poetry update for each example, Robb added a small script in the [releasing doc](https://github.com/honeycombio/libhoney-py/blob/main/RELEASING.md) to simplify the process.

**Comment by jharley** [comment]
> Looks like we need to run a poetry update for each example

If you mean the `for dir in [...]` one liner I had ran it. I just re-ran it with no changes:

```sh
$ for dir in `find ./ -type d -depth 1`; do (cd $dir && poetry update); done
Updating dependencies
Resolving dependencies... (2.7s)

No dependencies to install or update
Updating dependencies
Resolving dependencies... (1.9s)

No dependencies to install or update
Updating dependencies
Resolving dependencies... (1.9s)

No dependencies to install or update
Updating dependencies
Resolving dependencies... (1.9s)

No dependencies to install or update
Updating dependencies
Resolving dependencies... (1.9s)

No dependencies to install or update
```

**Comment by JamieDanielson** [comment]
> > Looks like we need to run a poetry update for each example
> 
> If you mean the `for dir in [...]` one liner I had ran it. I just re-ran it with no changes:
> 
> ```shell
> $ for dir in `find ./ -type d -depth 1`; do (cd $dir && poetry update); done
> Updating dependencies
> Resolving dependencies... (2.7s)
> 
> No dependencies to install or update
> Updating dependencies
> Resolving dependencies... (1.9s)
> 
> No dependencies to install or update
> Updating dependencies
> Resolving dependencies... (1.9s)
> 
> No dependencies to install or update
> Updating dependencies
> Resolving dependencies... (1.9s)
> 
> No dependencies to install or update
> Updating dependencies
> Resolving dependencies... (1.9s)
> 
> No dependencies to install or update
> ```

Oh well excellent thank for you confirming! ❤️

**Review by JamieDanielson — APPROVED** [review]

**Comment by JamieDanielson** [comment]
Also LMK if you are planning to release all the way through or are looking for us to finish the tag and GitHub release 👍  Thanks again!

**Comment by jharley** [comment]
> Also LMK if you are planning to release all the way through or are looking for us to finish the tag and GitHub release 👍 Thanks again!

Thank ya! I'm happy to tag and release 🚀

**Pull request merged**